lib/Backends/CPU/GlowJIT.h

```diff
@@ -46,6 +46,9 @@ class GlowJIT {
   const DataLayout DL_;
   RTDyldObjectLinkingLayer objectLayer_;
   IRCompileLayer<decltype(objectLayer_), SimpleCompiler> compileLayer_;
+  /// The listener for debugger events. It is used to provide debuggers with the
+  /// information about JITted code.
+  llvm::JITEventListener *dbgRegistrationListener = nullptr;
```

Comment:
Could this be a member of `NotifyLoadedFunctor` instead?  Also, nit: end member name with _.

Reply:
Done